### PR TITLE
a connector for Andreas

### PR DESCRIPTION
**Pull request by ivy-rew**
simple API, though with no official openapi.json ... so this one has been crafted swith saggers-only tooling

### CODE REVIEW

File: market/genderize/meta.json
Comment (alexsuter) [inline market/genderize/meta.json:10]: I think, if we commit the open api spec to this repository, we may should point it then relative.
(+) makes life easier for the maintainer of this repository. you can change the folder structure.
(+) makes life easier for connector provider, you don't need to know the "end" url.
(-) need to implement a relative path resolver in the open-api-rest-client installer.

good for now. in future I will implement a relative path resolver. anyway done for loading the logo.png.

**Review by alexsuter — APPROVED** [review]
nice!

**Comment by alexsuter** [comment]
all branches of this repository will be built-automatically.
https://jenkins.ivyteam.io/job/market/job/connector%252Fgenderize.io/

you did not make any mistake!
e.g. logo.png, meta.json, README.md is requiered

**Comment by alexsuter** [comment]
after merging to master. the master branch will be automatically built and the connector will be only available in the designer marketplace:
https://developer.axonivy.com/market-designer

in future, this will be also available in the "normal" market:
https://developer.axonivy.com/market

File: market/genderize/meta.json
Comment (ivy-rew) [reply market/genderize/meta.json:10]: Yes, we definitively will need a relative path resolver infrastructure.
For the future I could also imagine to create new repositories for each and every connector where we provide openapi.json ... with the idea to drive the openapi initiative in a way where others could contribute and enrich the specs. 
Like in this scenario here: I think for the operators of genderize.io it would be an easy fish to maintain this spec. And I think they would do that, if this repo does not live in our 'market' environment.

**Pull request merged**
